src/frontend/components/app-trees/app-trees.ts

```diff
@@ -1,21 +1,30 @@
-import {Component, Output, EventEmitter, Input}
-  from '@angular/core';
-import {NgClass} from '@angular/common';
+import {
+  Component,
+  Output,
+  EventEmitter,
+  Input
+} from '@angular/core';
 import TabMenu from '../tab-menu/tab-menu';
 import {TreeView} from '../tree-view/tree-view';
 import {RouterTree} from '../router-tree/router-tree';
 
 @Component({
   selector: 'bt-app-trees',
   directives: [TabMenu, TreeView, RouterTree],
-  inputs: ['tree', 'routerTree', 'selectedTabIndex',
-    'selectedNode', 'changedNodes', 'closedNodes'],
   templateUrl:
     '/src/frontend/components/app-trees/app-trees.html'
 })
 export default class AppTrees {
 
   @Input() theme: string;
+  @Input() tree: any;
+  @Input() routerTree: any;
+  @Input() selectedTabIndex: any;
```

Comment:
@ericjim @sumitarora that's why I'd like us all to use types more liberally that we are currently. This is a separate thing to address, out of the scope of this PR (unless you just want to add `:number`